app/uk/gov/hmrc/uknwauthcheckerapi/models/AuthorisationsResponse.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.uknwauthcheckerapi.models
+
+import play.api.libs.json.{Json, OFormat}
+
+case class AuthorisationsResponse(date: String, eoris: Array[EISResult])
+
+object AuthorisationsResponse {
+  implicit val format: OFormat[AuthorisationsResponse] = Json.format[AuthorisationsResponse]
+}
+
+case class EISResult(eori: String, authorised: Boolean)
```

Comment:
Pascal case is the standard naming convention

Reply:
Renamed the class name.